WORKSPACE

```diff
@@ -25,6 +25,26 @@ load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 load("//common:deps.bzl", "rules_python")
 rules_python()
 
+load("@graknlabs_bazel_distribution//maven:deps.bzl", maven_deps = "deps", "maven_artifacts_with_versions")
```

Comment:
Still not elegant enough I think. Can we aim for this:

```
# Load @rules_python
load("//common:deps.bzl", "rules_python")
rules_python()

# Load @io_bazel_rules_kotlin
load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kotlin_repositories", "kt_register_toolchains")
kotlin_repositories()
kt_register_toolchains()

# Load @graknlabs_bazel_distribution_pip
load("//pip:deps.bzl", pip_deps = "deps")
pip_deps()

# Load maven dependencies
load("//maven:deps.bzl", maven_deps = "deps", "maven_artifacts")
maven_deps()
```

Reply:
Can we make the call to `rules_jvm_external` happen within `maven_deps`?

Reply:
@lolski 

`io_bazel_rules_kotlin` has to come after `maven_deps` because `maven_deps` declares `http_archive` that loads Kotlin rules. If we're willing to split Kotlin rules out of `maven/deps.bzl`, then we might be able to do it.

>Can we make the call to rules_jvm_external happen within maven_deps?

Nope since we cannot `load` in macros

Reply:
`rules_kotlin` and `rules_jvm_external` should be in `//common:deps.bzl` together with `rules_python`. `//maven:deps` should only contain the list of maven artifacts that we want to bring in

Reply:
@lolski addressed in https://github.com/graknlabs/bazel-distribution/pull/293/commits/cf53380ae75a75ad6b95607a1e0862dff963f836